### PR TITLE
Poll support

### PR DESCRIPTION
**Pull request by faissaloux**
In this PR I've added poll support.

# Usage

```php
use NotificationChannels\Telegram\TelegramPoll;

TelegramPoll::create()
            ->to($notifiable)
            ->question($question)
            ->choices($choices); // array $choices
```

### CODE REVIEW

File: src/TelegramPoll.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    /**
+     * Poll options.
+     *
+     * @param  array  $options
+     *
+     * @return $this
+     */
+    public function options(array $options): self
```
Comment (irazasyed) [inline src/TelegramPoll.php:56]: This conflicts with the shared logic trait [here](https://github.com/laravel-notification-channels/telegram/blob/master/src/Traits/HasSharedLogic.php#L120).

**Comment by irazasyed** [comment]
Thanks for the PR. It looks good and is a nice addition.

Please look into the comment I made above and also, please add the docs to the Readme.

File: src/TelegramPoll.php (hunk 1 above)
Comment (faissaloux) [reply src/TelegramPoll.php:56]: Any rename suggestion for this!

Comment (irazasyed) [reply src/TelegramPoll.php:56]: How about "Choices" ?

Comment (faissaloux) [reply src/TelegramPoll.php:56]: Great, I'll rename it

**Comment by faissaloux** [comment]
Screenshot from 2021-11-23 22-53-50

**Comment by irazasyed** [comment]
Thanks 👍

**Comment by faissaloux** [comment]
Happy to help 😄

**Pull request merged**